src/main/java/org/threadly/util/ExceptionUtils.java

```diff
@@ -58,10 +58,24 @@ public static void setInheritableExceptionHandler(ExceptionHandler exceptionHand
   public static void setDefaultExceptionHandler(ExceptionHandler exceptionHandler) {
     defaultExceptionHandler = exceptionHandler;
   }
+
+  /**
+   * Gets the thread local {@link ExceptionHandler} if one is set, or {@code null} if none 
+   * is set.  Since {@link #getExceptionHandler()} prioritizes to the thread local handler, this can 
+   * be used to get a reference to the current handler before changing the thread local handler to 
+   * ensure that {@link #getExceptionHandler()} (and down stream use like 
+   * {@link #handleException(Throwable)}) invoke a handler of your choosing.  Once done you can then 
+   * choose to reset the original handler with the one returned from this invocation.
+   * 
+   * @return Thread local ExceptionHandler, or {@code null} if none is set
+   */
+  public static ExceptionHandler getThreadLocalExceptionHandler() {
```

Comment:
This ended up not being needed.  I originally had this because I was trying not to have "NoThread" implementations of the task wrappers.  Ultimately I was forced to in order to handle the non-system time behavior which could exist in the "TestableScheduler".

Before that happened in NoThreadScheduler.tick() I was actually getting the thread local ExceptionHandler, replacing it with my own (so I could handle exceptions that happened inside tasks, since the AbstractPriorityScheduler uses ExceptionUtils.runRunnable()).  Then after I was done I would set the thread local instance back to its original state.

Because that seemed like an important behavior I left this in, since it may be necessary from someone else.